cmake/ctest/drivers/geminga/TrilinosCTestDriverCore.geminga.gcc-cuda.cmake

```diff
@@ -71,15 +71,15 @@ MACRO(TRILINOS_SYSTEM_SPECIFIC_CTEST_DRIVER)
     SET(BUILD_DIR_NAME ${COMM_TYPE}-${BUILD_TYPE}_${BUILD_NAME_DETAILS})
   ENDIF()
 
-  SET(Trilinos_REPOSITORY_LOCATION_NIGHTLY_DEFAULT "git@github.com:muelu/Trilinos.git")
+  SET(Trilinos_REPOSITORY_LOCATION_NIGHTLY_DEFAULT "https://github.com/muelu/Trilinos.git")
 
   SET(CTEST_DASHBOARD_ROOT  "${TRILINOS_CMAKE_DIR}/../../${BUILD_DIR_NAME}" )
   SET(CTEST_NOTES_FILES     "${CTEST_SCRIPT_DIRECTORY}/${CTEST_SCRIPT_NAME}" )
   SET(CTEST_BUILD_FLAGS     "-j35 -i" )
 
   SET_DEFAULT(CTEST_PARALLEL_LEVEL                  "35" )
   SET_DEFAULT(Trilinos_ENABLE_SECONDARY_TESTED_CODE ON)
-  SET(Trilinos_CTEST_DO_ALL_AT_ONCE FALSE)
+  SET(Trilinos_CTEST_DO_ALL_AT_ONCE ON)
```

Comment:
what does this do?

Reply:
It runs all packages together instead of one by one. It should be somewhat faster.

Reply:
Found [this documentation](https://www.docs.trilinos.org/files/TrilinosBuildReference.html#common-options-and-use-cases-for-the-dashboard-target).  This might swamp the test machine?

Reply:
https://tribitspub.github.io/TriBITS/users_guide/index.html#project-name-ctest-do-all-at-once

Reply:
https://tribitspub.github.io/TriBITS/users_guide/index.html#all-at-once-versus-package-by-package-mode-tribits-ctest-driver

Reply:
@bartlettroscoe Thanks!  I now remember that you explained this to me once.

Reply:
> This might swamp the test machine?

It should not.  You can control the parallel build and test level to ensure the max RAM usage is not too high.  No one uses the package-by-package mode anymore.